### PR TITLE
Updates support options, adding StackOverflow, fixes #872

### LINKED ISSUE #872: Add mention of Stack Overflow to docs
**Describe the bug**
Documentation doesn't mention Stack Overflow. 

**To Reproduce**
Search docs! 

**Expected behavior**
Clear instructions about support options!

https://ddev.readthedocs.io/en/latest/users/troubleshooting/
https://ddev.readthedocs.io/en/latest/#support

### PR DESCRIPTION
**Pull request by andrewfrench**
## The Problem/Issue/Bug:
OP #872 mentions adding StackOverflow links to docs, this PR adds those references.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,8 @@ If you're having trouble using ddev, please use these resources to get help:
 2. Review [Stack Overflow DDEV-Local questions and answers](https://stackoverflow.com/tags/ddev) (or ask a question there! We get notified when you ask.)
 3. The [ddev issue queue](https://github.com/drud/ddev/issues) may have an issue related to your problem.
 4. For suspected bugs or feature requests, [file an issue](https://github.com/drud/ddev/issues/new).
-5. We have slack channels (#ddev) in [Drupal Slack](https://www.drupal.org/slack) and [TYPO3 Slack](https://forger.typo3.com/slack). We and the rest of the community try to be very responsive.
+5. #ddev channel in [Drupal Slack](https://www.drupal.org/slack) and [TYPO3 Slack](https://forger.typo3.com/slack) for interactive, immediate community support
```
Comment (rickmanelius) [inline README.md]: FWIW, Slack offers the ability to direct link to a room. For example
* Drupal: https://drupal.slack.com/messages/C5TQRQZRR
* TYPO3: https://typo3.slack.com/messages/C8TRNQ601

File: docs/index.md [hunk 2]
```diff
@@ -68,8 +68,8 @@ Note that third party packaging is encouraged, but only supported on a best-effo
 The DDEV project is committed to supporting [Semantic Version 2.0.0](https://semver.org/). Additional context on this decision can be read in [Ensure ddev is properly utilizing Semantic Versioning](https://github.com/drud/ddev/issues/352).
 
 ## Support
-If you've encountered trouble using ddev, please use these resources to get help with your issue:
 
-1. Please review the [ddev Documentation](https://ddev.readthedocs.io) to ensure your question isn't answered there.
-2. Review the [ddev issue queue](https://github.com/drud/ddev/issues) to see if an issue similar to yours already exists.
-3. If you've exhausted these options and still need help, please [file an issue](https://github.com/drud/ddev/issues/new) following the pre-populated guidelines and our [Contributing Guidelines](https://github.com/drud/ddev/blob/master/CONTRIBUTING.md) as best as possible.
+- [ddev Documentation](https://ddev.readthedocs.io)
```
Comment (rickmanelius) [inline docs/index.md]: Not sure it's necessary to self link considering we're in the docs, but I also see the value in having a single block of markdown we can copy/paste elsewhere and include other places.

**Comment by rfay** [comment]
@andrewfrench We often do as Rick has done here, approving a PR with some suggestions, which means that we're asking for simple followup commits to be pushed in by the author, but they don't need further review.

File: README.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ If you're having trouble using ddev, please use these resources to get help:
 2. Review [Stack Overflow DDEV-Local questions and answers](https://stackoverflow.com/tags/ddev) (or ask a question there! We get notified when you ask.)
 3. The [ddev issue queue](https://github.com/drud/ddev/issues) may have an issue related to your problem.
 4. For suspected bugs or feature requests, [file an issue](https://github.com/drud/ddev/issues/new).
-5. #ddev channel in [Drupal Slack](https://www.drupal.org/slack) and [TYPO3 Slack](https://forger.typo3.com/slack) for interactive, immediate community support
+5. #ddev channel in [Drupal Slack](https://drupal.slack.com/messages/C5TQRQZRR) and [TYPO3 Slack](https://typo3.slack.com/messages/C8TRNQ601) for interactive, immediate community support
```
Comment (rfay) [inline README.md]: Wow, I didn't even know that you could do this.

**Pull request merged**